### ISSUE TITLE
Content search seems not working correctly

### ISSUE THREAD

**Issue by ricopin411029** [g1eny0ung/hugo-theme-dream#219]
Hi 
Thank you so much for creating this beautiful theme :). I love it! 
Just one thing I came across is, it seems search box doesn't work correctly. 
While I just enabled search in config.toml as follows, it doesn't work and I see the error `Uncaught TypeError: $(...).search is not a function`. 
Can you give any idea on what cause this and how to fix?  
P.S. It also happens on your blog --> https://g1eny0ung.site/. 

Here's my detailed settings:

```
# config.toml
[outputs]
  home = ["HTML", "JSON"]

[params]
  enableSearch = true
```
There's no other customization, I just use default settings.  Stack trace of this error is as follows:  

```
(index):463 Uncaught TypeError: $(...).search is not a function
    at Object.success ((index):463)
    at c (jquery@3.6.0:2)
    at Object.fireWith [as resolveWith] (jquery@3.6.0:2)
    at l (jquery@3.6.0:2)
    at XMLHttpRequest.<anonymous> (jquery@3.6.0:2)
```

My environment is as follows:  
Browser: Chrome 91.0.4472.77 (Official Build) (x86_64) 
** But I see the same error with the Firefox 88.0.1 as well. 
OS: macOS Version 11.3.1 (Build 20E241)
Hugo: hugo version output as follows
hugo v0.83.1+extended darwin/amd64 BuildDate=unknown

and checkout your master branch.  

Any advice or comments are appreciated!  
Best,

Rico.

**Comment by g1eny0ung** [comment]
Thx for the feedback. It's a stupid problem 🤦🏽‍♂️ and already fixed in 57130f9523d6c320f893e762031214c57978c897.

**Comment by ricopin411029** [comment]
Thank you so much for prompt reply! It works!! 
Thanks again, wish this beautiful theme success! 

Best,
Rico.
